perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -64,6 +64,7 @@ class GCPManagedRelationalDb(managed_relational_db.BaseManagedRelationalDb):
   # These are the constants that should be specified in GCP's cloud SQL command.
   GCP_PRICING_PLAN = 'PACKAGE'
   MYSQL_DEFAULT_PORT = 3306
+  VALID_POSTGRES_MACHINE_TYPES = ['db-f1-micro', 'db-g1-small']
```

Comment:
[Documentation](https://cloud.google.com/sql/pricing#pg-pricing) says that shared-cored and custom machines are available to PostGres.

I did try it out manually first. See attached screenshot.
![screenshot from 2017-06-27 09 45 27](https://user-images.githubusercontent.com/10179903/27599374-6e3baeb4-5b1d-11e7-9c05-c4e9c6f335d9.png)